src/main/features/claim/views/check-and-send.njk

```diff
@@ -42,6 +42,9 @@
     }}
 
     {{ row('Email', draftClaim.defendant.email.address, paths.defendantEmailPage.uri, bold = true) }}
+    {% if draftClaim.defendant.mobilePhone %}
+      {{ row('Contact number (optional)', draftClaim.defendant.mobilePhone.number, paths.defendantPhonePage.uri, bold = true) }}
+    {% endif %}
```

Comment:
It should still be shown in the check-and-send page if it's absent, so the user sees that it hasn't been entered and gets the link to go and change it.

Reply:
sorted it.